src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -11,11 +11,12 @@
 #include "routes_conn.h"
 #include "oper_db_client/sl_oper_db_client.h"
 #include "../../../picomsg/src/picomsg.h"
-
+#include "oper_db_writer.h"
```

Comment:
Please keep the current structure of includes:
1. Internal includes with ""
2. [empty line]
3. Internal includes with <>
4. [empty line]
5. External includes with <>